packages/components/src/text-input/README.md

```diff
@@ -0,0 +1,232 @@
+# TextInput
+
+`TextInput` is a form component that users can enter content into.
+
+## Table of contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [Usage](#usage)
+- [Props](#props)
+- [See Also](#see-also)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+<!-- Automatically Generated. DO NOT EDIT THIS FILE. -->
+<!-- Instead, edit packages/website/src/docs/components/forms/textinput.mdx -->
+
+<!-- props -->
+
+<!-- Automatically Generated -->
```

Comment:
We should remove this from the README as well as add the `experimental` callout that we've added to the other experimental components.